src/analytics/event.ts

```diff
@@ -1,35 +1,113 @@
+import { CardType } from '@/components/cards/GenericCard';
```

Comment:
leaving a reminder for myself to talk to @estrattonbailey about how this file should be structured

Reply:
Ah yeah, sorry, we need to rebase this. Docs here https://github.com/rainbow-me/rainbow/tree/develop/src/analytics